linty_fresh/linters/xcodebuild.py

```diff
@@ -4,10 +4,12 @@
 
 from linty_fresh.problem import Problem
 
-XCODEBUILD_LINE_REGEX = re.compile(r'(?P<path>[^:]*):'
-                                   r'((?P<line>\d*)|(?P<reference>[^:]*)):'
-                                   r'(?:(?P<column>\d*):)?\s*(?P<level>\w*):'
-                                   r'\s*(?P<message>.*)')
+XCODEBUILD_LINE_REGEX = re.compile(
+    r'(?P<path>[^:]*):'
+    r'((?P<line>\d*)|(?P<reference>[^:]*)):'
+    r'(?:(?P<column>\d*):)?\s*(?P<level>(error|warning|note)):'
```

Comment:
The only change besides indentation is this level regex

Reply:
1. Are we sure there aren't other level names? Like `info`, `warn`?
2. I'm thinking this regex be compiled case insensitive, or at least these levels: `((?i)error|warning|note)`

Reply:
Could be more, I can definitely add those 2, any others in mind?

Reply:
The number of uses of `*` instead of `+` in this regex bothers me 😂

Reply:
Done

Reply:
Nothing else that comes to mind.